pontoon/base/templatetags/helpers.py

```diff
@@ -280,23 +280,27 @@ def as_simple_translation(source):
     if type(translation_ast) == ast.Junk:
         return source
 
+    # Value: use entire ast
     if translation_ast.value:
-        # Strings with variants: display first variant
-        try:
-            elements = translation_ast.value.elements
-            variants = elements[0].expression.variants
-            return _serialize_elements(variants[0].value.elements)
+        tree = translation_ast
 
-        # Simple strings
-        except (AttributeError, IndexError):
-            return _serialize_elements(translation_ast.value.elements)
+    # Attributes: use first attribute
+    elif translation_ast.attributes:
+        tree = translation_ast.attributes[0]
 
+    # This should never happen! Return fallback.
     else:
-        # Strings with attributes: display first attribute
-        try:
-            attributes = translation_ast.attributes
-            return _serialize_elements(attributes[0].value.elements)
-
-        # All other strings: display unchanged
-        except (AttributeError, IndexError):
-            return source
+        return source
```

Comment:
If this should never happen, maybe it would be worth raising an Exception instead? This way, if it does actually happen, we'll know.